README.md

```diff
@@ -432,7 +432,23 @@ Once selenium server is running, in a seperate terminal you should be able to ru
 ```bash
 npm run tests:e2e
 ```
- 
+
+##### Mocking the API for e2e tests
+The backend API is a pass-through to a soap service which is not accessible during testing.
+To facilitate complete end-to-end tests, the API server can be placed into mock mode by setting
+the `HCA_MOCK_API=1` environment variable.
```

Comment:
There are a few ways to do it, but I'm always a proponent of just passing it directly to npm via the command line:

`HCA_MOCK_API=1 npm run watch`